ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -77,8 +89,9 @@ node('caasp-team-private-integration') {
            }
            if (pr_repo_label != null) {
                def branch_name = pr_repo_label.name.split(":")[1]
-               branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/${repo_version}:/Branches:/${branch_name}/SLE_15_SP1"
-
+               branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/${repo_version}:/Branches:/${branch_name}/SLE_15_SP2"
```

Comment:
These configs here will not work on v4, but I would not bother that match about it. I think we can live without `ci-repo:branch` feature in CaaSPv4, don't we?